### ISSUE TITLE
Do not only search with #beginsWith:

### ISSUE THREAD

**Issue by guillep** [guillep/complishon#11]
Right now we only search selectors that `#beginsWith:` the partially typed token.
It would maybe make sense to, with less priority, also search those that include the token anywhere in the string.

**Comment by Ducasse** [comment]
In fact the beginsWith: works well in forward writing. 
Often I modify an existing word and in that case alternative solutions may fit. 
So we could check that there is nothing on the right of the cursor -> beginsWith:
else another match.

**Comment by guillep** [comment]
Yes, maybe, that made me think I also want this: https://github.com/guillep/complishon/issues/12
